core/execution/equity_shares.go

```diff
@@ -67,8 +67,8 @@ func (es *EquityShares) OpeningAuctionEnded() {
 // for every LP during opening auction.
 func (es *EquityShares) setOpeningAuctionAVG() {
 	// now set average entry valuation for all of them.
-	factor := num.DecimalFromFloat(1.0)
-	if !es.totalVStake.IsZero() && !es.totalPStake.IsZero() {
+	factor := num.DecimalFromFloat(1)
+	if !es.totalPStake.IsZero() && !es.totalVStake.IsZero() {
 		factor = es.totalPStake.Div(es.totalVStake)
```

Comment:
Not sure why this is happening? Can't see anything in the spec about updating AEV values at the end of an opening auction?